src/Draco.Extension.VsCode/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "ES2020",
+    "outDir": "out",
+    "lib": [
+      "ES2020"
+    ],
+    "sourceMap": true,
+    "rootDir": "src",
+    "strict": true   /* enable all strict type-checking options */
+    /* Additional Checks */
```

Comment:
personal taste: no implicit any.
https://www.typescriptlang.org/tsconfig#noImplicitAny

Reply:
I don't really use TS so I'll trust you on this.